platform/darwin/src/MGLRendererConfiguration.h

```diff
@@ -26,11 +26,19 @@ MGL_EXPORT
 /** The name of the font family to use for client-side text rendering.
 
  Currently only used for CJK glyphs. Changing this at run time is not currently
- supported. Enable client-side rendering of CJK glyphs by setting
- `MGLIdeographicFontFamilyName` in your containing app's Info.plist to a value
- which will be available at run time. Default font for local ideograph font family
- is "PingFang". */
-@property (nonatomic, readonly) std::string localFontFamilyName;
+ supported. By default, client-side rendering is enabled for CJK glyphs using Apple
+ system default font.
+ 
+ Set `MGLIdeographicFontFamilyName` in your containing app's Info.plist to a string
```

Comment:
AFAIK `MGLRendererConfiguration.h` is a project header;  we should also add this documentation to `Info.plist Keys.md`